compiler/testData/diagnostics/tests/modifiers/const/applicability.fir.kt

```diff
@@ -12,12 +12,12 @@ object A {
     const val inObject: Int = 4
 }
 
-class B(const val constructor: Int = 5)
+class B(<!CONST_VAL_NOT_TOP_LEVEL_OR_OBJECT!>const<!> val constructor: Int = 5)
```

Comment:
This case is a bit specific because `constructor` is parameter and property simultaneously, and looks like the corresponding `FirProperty` does not have `isConst` set. Will try to fix this behavior.

Reply:
Fixed